cypress.config.ts

```diff
@@ -1,8 +1,26 @@
 import { defineConfig } from 'cypress';
+import * as fs from 'fs';
 
 export default defineConfig({
   e2e: {
     baseUrl: 'https://the-internet.herokuapp.com/',
-    specPattern: 'cypress/tests/*.cy.ts'
+    specPattern: 'cypress/tests/*.cy.ts',
+    setupNodeEvents(on, config) {
+      on('task', {
+        checkFileExists(filePath: string) {
+          let something = fs.existsSync(filePath);
```

Comment:
Restructure this to just return the existsSync() result (no need for variable assignment)